omod/src/main/webapp/pages/visit/todaysVisits.gsp

```diff
@@ -35,6 +36,13 @@
                             "visit": visit.uuid ])}">
                         ${ ui.format(visit.patient) }
                     </a>
+                </td>
+                 <td>
+                    <a href="${ ui.pageLink("pihcore", "visit/visit#/print", [
```

Comment:
This calls the "print summary" function for that visit, I believe. Is that what was intended, or do you just want the link to open that visit for the patient? For just opening the visit you'll want to copy the link from the one of the first two columns, not the "print summary" column.